### PR TITLE
WIP: Add FreeBSD 13 for node_exporter and server

### PR DESCRIPTION
**Pull request by sebastianrakel**
I added some default settings in Data to be working on FreeBSD 13

### CODE REVIEW

**Comment by ghoneycutt** [comment]
Thanks for the contribution @sebastianrakel 

@smortex Thought this might be something you would be interested in.

File: data/FreeBSD.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+prometheus::init_style: 'none'
+prometheus::configname: 'prometheus.yml'
+prometheus::server::init_style: 'none'
+prometheus::version: '2.30.0'
+prometheus::install_method: 'package'
+prometheus::bin_dir: '/usr/bin'
```
Comment (smortex) [inline data/FreeBSD.yaml:7]: On FreeBSD I would expect extra software to be installed in `/usr/local/bin`

File: data/FreeBSD.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+prometheus::env_file_path: '/etc/defaults'
```
Comment (smortex) [inline data/FreeBSD.yaml:8]: Likewise I would expect configuration to be under `/usr/local/etc`

**Review by smortex — COMMENTED** [review]
Hey, not a user of this module nor prometheus so maybe I am completely off… a few things look odd though :shrug:.

**Comment by TheMeier** [comment]
Since nothing happened here for 3 years I am closing this. @sebastianrakel  feel free to re-open.

**Pull request closed without merging**